packages/hawtio/src/plugins/shared/operations/operation.ts

```diff
@@ -23,13 +25,21 @@ function addOperation(
   name: string,
   op: IJmxOperation,
 ): void {
-  const operation = new Operation(
-    name,
-    op.args.map(arg => new OperationArgument(arg.name, arg.type, arg.desc)),
-    op.desc,
-    op.ret,
-    op.canInvoke,
-  )
+  const args: OperationArgument[] = []
+  for (const arg of op.args) {
+    let argObj = arg
+    if (!isObject(arg)) {
```

Comment:
It's still not clear to me why this check is needed. The arg type is defined like this:
https://github.com/hawtio/jolokia.js/blob/dc6bb0c8c8c1f1904b58630c8a1aab9e69ac2f36/index.d.ts#L65-L69
Maybe you can provide us with an example of Jolokia requests that are irregular to the defined type, then we can modify the type definition accordingly. Or it could be a bug in Jolokia that should rather be raised to Jolokia project.